src/Resources/templates/trait.php.twig

```diff
@@ -12,7 +12,7 @@ use {{ use_statement.name}} as {{ use_statement.alias }};
  * This is the guy that the main class needs to require, and ensures that everything is glued
  * together
  */
-trait {{ class_name }}Traits
+trait {{ class_name }}Trait
```

Comment:
Why is this being changed? It implies a big BC break if this code is doing what I think it's doing